src/Thermal/RadwsEOS.cpp

```diff
@@ -106,7 +109,9 @@ FLOAT Radws<ndim>::SpecificInternalEnergy(Particle<ndim> &part)
 template <int ndim>
 FLOAT Radws<ndim>::Temperature(Particle<ndim> &part)
 {
-  return (part.gamma - 1.0)*part.u*part.mu_bar;
+  FLOAT gamma = opacity_table->GetGamma(part);
```

Comment:
I think this is also incorrect.

Reply:
Ok, I'm happy with this as long as mu_bar as boltzmann's constant absorbed into it. Is this correct?

Reply:
Ok, actually this is fine: mu_bar should be something like 2.4 at low temperatures. I'd forgotten we use a strange definition for the temperature.